Makefile

```diff
@@ -265,6 +273,31 @@ test-unit-with-coverage:
 		go test $(shell GOCACHE="$(GOCACHE)" go list ./...|grep -v e2e) $(GOCOV_FLAGS) -v -mod vendor $(TEST_EXTRA_ARGS)
 	$(Q)GOCACHE=$(GOCACHE) go tool cover -func=$(GOCOV_FILE)
 
+.PHONY: test-acceptance-setup
+## Setup the environment for the acceptance tests
+ifeq ($(TEST_ACCEPTANCE_START_SBO), local)
+test-acceptance-setup:
+	$(Q)echo "Starting local SBO instance"
+	$(eval TEST_ACCEPTANCE_SBO_STARTED := $(shell OPERATOR_NAMESPACE="$(TEST_NAMESPACE)" ZAP_FLAGS="$(ZAP_FLAGS)" ./hack/deploy-sbo-local.sh))
+else ifeq ($(TEST_ACCEPTANCE_START_SBO), operator-hub)
+test-acceptance-setup:
+	$(eval TEST_ACCEPTANCE_SBO_STARTED := $(shell ./hack/deploy-sbo-operator-hub.sh))
+endif
+
+.PHONY: set-test-namespace
+set-test-namespace: get-test-namespace
+	$(Q)oc project $(TEST_NAMESPACE)
+
+.PHONY: test-acceptance
+## Runs acceptance tests
+test-acceptance: e2e-setup set-test-namespace deploy-clean deploy-rbac deploy-crds test-acceptance-setup
```

Comment:
@pratikjagrut The acceptance tests re-use the very same test namespace mechanic as for the old e2e tests.
In addition, to be able to run the tests with the current state of the repository, it uses a local instance of the SBO started via `operator-sdk run --local ...`, similarly as in case of the old e2e tests, and so all the pre-run targets apply.

And as for the `set-test-namespace` target, as mentioned below, it does not create a namespace, it is only used to sed the `oc` tool with the test namespace in case the `oc` tool does not have it set (sometimes it happens, especially when switching clusters).

So, the `set-test-namespace` target is used to ensure the `oc` tool is set for the namespace properly, not to create a new one.

Reply:
Okay.